sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/tests/Azure.Monitor.OpenTelemetry.Exporter.Demo.Tracing/Program.cs

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Diagnostics.Metrics;
+using Microsoft.Extensions.Logging;
+using OpenTelemetry;
+using OpenTelemetry.Resources;
+using OpenTelemetry.Metrics;
+using OpenTelemetry.Trace;
+
+namespace Azure.Monitor.OpenTelemetry.Exporter.Demo.Tracing
+{
+    public class Program
+    {
+        private const string ConnectionString = "InstrumentationKey=00000000-0000-0000-0000-000000000000";
+        private static readonly ActivitySource source = new ActivitySource("MyCompany.MyProduct.MyLibrary");
+        private static readonly Meter MyMeter = new("MyCompany.MyProduct.MyLibrary", "1.0");
+
+        public static void Main()
+        {
+            GenerateTraces();
+            GenerateLogs();
+            GenerateMetrics();
+            Dispose();
```

Comment:
Probably no need to Dispose explicitly here, I think even the official .NET example relies on GC to do the clean-up (in a non-deterministic way).

Reply:
This can be a good example to follow https://docs.microsoft.com/en-us/dotnet/core/diagnostics/metrics-collection#configure-the-example-application-to-use-opentelemetrys-prometheus-exporter.